business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -217,6 +217,7 @@
             <configuration>
               <excludes>
                 <exclude>**/ProjectLibraryIntegrationTest.java</exclude>
+                <exclude>**/StandalonePerspectivesIntegrationTest.java</exclude>
```

Comment:
This test doesn't make sense for business-monitoring, as that war doesn't have authoring stuff included.